vendor/modules.txt

```diff
@@ -239,6 +239,8 @@ github.com/heptiolabs/healthcheck
 # github.com/imdario/mergo v0.3.6
 ## explicit
 github.com/imdario/mergo
+# github.com/itchyny/json2yaml v0.1.4
```

Comment:
I don't think this should be here either.

Reply:
CI build complains🤔

go: inconsistent vendoring in /go/src/agones.dev/agones:
	github.com/itchyny/json2yaml@v0.1.4: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt

	To ignore the vendor directory, use -mod=readonly or -mod=mod.
	To sync the vendor directory, run:
		go mod vendor

Reply:
Alright, if you say so, there must be a reason behind it. I've reverted the changes in go.sum, go.mod, and vendor/modules.txt

Reply:
Oh no, If I run the code the go.sum and go.mod files are automatically updated. I guess this is because of `-mod=mod`?
```
gen-embedded-openapi: ensure-build-image
	docker run --rm $(common_mounts) --workdir=$(mount_path) $(DOCKER_RUN_ARGS) $(build_tag) \
		go run -mod=mod build/scripts/k8s-export-openapi/main.go
```

However, the code is failing without `-mod=mod`. What could be the solution?

Reply:
Maybe a silly question - what happens if we do:

```
gen-embedded-openapi: ensure-build-image
	docker run --rm $(common_mounts) --workdir=$(mount_path)/build/scripts/k8s-export-openapi $(DOCKER_RUN_ARGS) $(build_tag) \
		go run -mod=mod ./main.go
```

(Might not need the `-mod=mod` either at that point?)

Reply:
I believe that if the `workdir` changes, the paths to other files in the code will also need to be modified.

Reply:
Try it? I expect the script will fail, but we'll see if it updates the other go.mod and go.sum files -- and if it doesn't, then we know making the path changes worth it.

Reply:
The good news is - with the agones/go.mod and agones/go.sum changes reverted - CI is passing again - so that's at least a step in the right direction!

Reply:
Script passed with the below modifications:

tmpDir := "../../tmp"
boilerplateContent, err := os.ReadFile("../../boilerplate.yaml.txt")
outputPath := filepath.Join("../../../install/helm/agones/templates/crds/k8s", "_"+filename+".yaml")

Reply:
Love it 😄